From 47fa53a3c34ae5aa5eccf3bc92922b15b09a423f Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Diego=20Elio=20'Flameeyes'=20Petten=C3=B2?= <flameeyes@gmail.com>
Date: Thu, 29 Jul 2010 00:17:02 +0200
Subject: [PATCH] Make sure that the required paths are absolute.

Ruby 1.9.2 no longer adds the CWD to the list of searched paths.
---
 Rakefile |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/Rakefile b/Rakefile
index 29ef0e2..c4e9887 100644
--- a/Rakefile
+++ b/Rakefile
@@ -128,8 +128,8 @@ file GENERATED_TOKENIZER => "lib/nokogiri/css/tokenizer.rex" do |t|
   end
 end
 
-require 'tasks/test'
-require 'tasks/cross_compile'
+require File.dirname(__FILE__) + '/tasks/test'
+require File.dirname(__FILE__) + '/tasks/cross_compile'
 
 desc "set environment variables to build and/or test with debug options"
 task :debug do
-- 
1.7.2

